src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```
@@ -39,7 +39,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   -->
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and
                             '$(HasRuntimeOutput)' == 'true' and
-                            '$(OS)' == 'Windows_NT' and
+                            $([MSBuild]::IsOSPlatform('Windows') and
```

this is missing a parenthesis:

```suggestion
                            $([MSBuild]::IsOSPlatform('Windows')) and
```